scania-digital-design-system/tegel
# feat(tabs): allow custom left padding

## **Describe pull-request**  
Enable ability for users to pass a custom left padding value through a prop. 

## **Issue Linking:**  
- **Jira:** Add ticket number after `CDEP-3454`: [CDEP-](https://tegel.atlassian.net/browse/CDEP-3454)    

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to Storybook
2. Check in Inline Tabs and Navigation Tabs
3. Set the left padding prop, verify that it only affects the first element
